etna/collections/blocks.py

```diff
@@ -23,38 +10,16 @@ class PromotedPagesBlock(blocks.StructBlock):
 
     class Meta:
         template = "collections/blocks/promoted_pages.html"
-        help_text = "Block used promote external pages"
-        icon = "th-large"
-
-
-class TopicExplorerIndexBlock(blocks.StructBlock):
-    heading = blocks.CharBlock(max_length=100, default="Explore by topic")
-    page = blocks.PageChooserBlock(page_type="collections.TopicExplorerIndexPage")
-
-    class Meta:
-        template = "collections/blocks/topic_explorer.html"
-        help_text = "Outputs all topic child pages"
-        icon = "th-large"
-
-
-class TimePeriodExplorerIndexBlock(blocks.StructBlock):
-    heading = blocks.CharBlock(max_length=100, default="Explore by time period")
-    page = blocks.PageChooserBlock(page_type="collections.TimePeriodExplorerIndexPage")
-
-    class Meta:
-        template = "collections/blocks/time_period_explorer.html"
-        help_text = "Outputs all time period child pages"
+        help_text = "Block used to promote external pages"
         icon = "th-large"
 
 
 class ExplorerIndexPageStreamBlock(blocks.StreamBlock):
-    time_period_explorer_index = TimePeriodExplorerIndexBlock()
-    topic_explorer_index = TopicExplorerIndexBlock()
```

Comment:
Note: As you're deleting these classes, can we delete the templates too or are they used anyhwere?

Reply:
Hey @William-Blackie, the templates are used in another block so I shall leave them for now :)